webnode/config/webpack.config.prod.js

```diff
@@ -312,28 +223,12 @@ module.exports = {
         console.log(message);
       },
       minify: true,
-      // For unknown URLs, fallback to the index page
       navigateFallback: publicUrl + "/index.html",
       // Ignores URLs starting from /__ (useful for Firebase):
       // https://github.com/facebookincubator/create-react-app/issues/2237#issuecomment-302693219
       navigateFallbackWhitelist: [/^(?!\/__).*/],
       // Don't precache sourcemaps (they're large) and build asset manifest:
       staticFileGlobsIgnorePatterns: [/\.map$/, /asset-manifest\.json$/]
```

Comment:
Same with this file, and any others where comments got deleted.

Reply:
i think the variable name is obvious enough. I know that no everyone knows webpack, but i planned on writing a directions in the readme for how to use and modify. I think that would be much more helpful.